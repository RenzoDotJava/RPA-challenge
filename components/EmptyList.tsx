import { View, Text, StyleSheet, Platform } from 'react-native'

type EmptyListProps = {
  text: string
}

const EmptyList: React.FC<EmptyListProps> = ({text}) => {
  return (
    <View style={styles.container}>
      <Text style={styles.text}>{text}</Text>
    </View>
  )
}

const styles = StyleSheet.create({
	container: {
		flex: 1,
    marginTop: 15
	},
  text: {
    textAlign: "center",
    fontFamily: Platform.OS === "android" ? "serif" : "Arial",
    fontSize: 20,
    fontWeight: "500"
  }
});

export default EmptyList