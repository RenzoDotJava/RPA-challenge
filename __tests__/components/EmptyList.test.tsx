import { render, screen } from "@testing-library/react-native";
import EmptyList from "../../components/EmptyList";

const props = {
	text: 'text'
};

describe("Testing <PublicationTitle />", () => {
	it("Matching snapshot", () => {
		render(<EmptyList {...props} />);

		expect(screen).toMatchSnapshot();
	});

	it("Showing the props", () => {
		render(<EmptyList {...props} />);

		expect(screen.getByText(props.text)).toBeTruthy();
	});
});