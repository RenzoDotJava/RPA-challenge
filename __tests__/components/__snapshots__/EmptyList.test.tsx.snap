// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing <PublicationTitle /> Matching snapshot 1`] = `
<View
  style={
    {
      "flex": 1,
      "marginTop": 15,
    }
  }
>
  <Text
    style={
      {
        "fontFamily": "Arial",
        "fontSize": 20,
        "fontWeight": "500",
        "textAlign": "center",
      }
    }
  >
    text
  </Text>
</View>
`;
